### DISCUSSION TITLE
[Feature] List of all Stacks

### DISCUSSION THREAD

**Discussion by lukanowa** [immich-app/immich#8987] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think will be great to have menu/toolbar with list od all stacks.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Alternatively this could be implemented as a search filter.

**Comment by mmomjian** [comment]
If this search filter is added, I think it would be great to also add a filter to search for live photos as well. They should be similar to implement, based on the DB structure.

**Comment by pippo73** [comment]
I join the list of the interested ones :-)

**Comment by nickdollimount** [comment]
Bumping this. I'm a photographer and after a shoot, I might dump 500+ photos on my server. Those folders are included so as you can imagine, that chunk of pictures can be a pin to have just show up in the middle of my timeline. I've been using the stacking feature to just stack the chunk of photos to clean things up. I just stumbled around to see how I could find all my stacks and there doesn't appear to be. It would be really great for this feature to be added. I imagine biggest part of the functionality already exists to be able to show the stacks in the timeline, so I'd guess it would be fairly straightforward to create a filter for those and a link on the sidebar under **Library**.

**Comment by bondeabhijeet** [comment]
Same with me, I have been trying to manage my library of around 23k assets, and sometimes I simply forget some picture exists, simply because it had been stacked. It would be really useful to have a place or search utility that would help navigate through all the stacks.

**Comment by AMDFreak2006** [comment]
Thumbs up, I accidentally used the Stacking feature several times during duplicate finding works and without search function its nearly impossible to find them again to unstack them. Please implement this feature!

**Comment by ebgenius** [comment]
I am starting to enjoy creating fun stacks, rather than deleting all the duplicate photos in the duplicate utility page.
The problem is, I want to search and find again the stack I just created and there is no way to do it!
Bumped!
